backend/model/model.go

```diff
@@ -937,6 +938,10 @@ type ErrorGroup struct {
 	} `gorm:"-"`
 	FirstOccurrence *time.Time `gorm:"-"`
 	LastOccurrence  *time.Time `gorm:"-"`
+
+	// Represents the admins that have viewed this session.
+	ViewedByAdmins []Admin `json:"viewed_by_admins" gorm:"many2many:error_group_admins_views;"`
```

Comment:
What do you think about storing a timestamp on this table as well? I've wanted that a few times on the `session_admins_views` table and I'm guessing we'll want it at some point as well. Might require using a [custom join table](https://gorm.io/docs/many_to_many.html#Customize-JoinTable) but can still leverage the `many2many` helper.

Reply:
good point! we added that for session_admins_views in prod, but i'll make it more explicit here and also do the same for the error group views.